xml/ha_qdevice-qnetd.xml

```diff
@@ -0,0 +1,508 @@
+      -->
+      <para>
+         To get the quorum of your cluster, you can do so on one
+         of your cluster nodes as shown in <xref
+            linkend="ex-ha-qdevice-crm-corosync-status-quorum"/>.
+         It shows you the status of your &qdevice; components.
+      </para>
+
+      <example xml:id="ex-ha-qdevice-crm-corosync-status-quorum">
+         <title>Output Status of &qdevice;</title>
+         <screen>&prompt.root;<command>crm</command> corosync status quorum <co xml:id="co-quorum-cmd"/>
+ Quorum information
+------------------
+Date:             ...
+Quorum provider:  corosync_votequorum
+Nodes:            2 <co xml:id="co-quorum-nodesnumber"/>
+Node ID:          3232235777 <co xml:id="co-quorum-nodeid"/>
+Ring ID:          3232235777/8
+Quorate:          Yes <co xml:id="co-quorum-quorate"/>
+
+Votequorum information
+----------------------
+Expected votes:   3
+Highest expected: 3
+Total votes:      3
+Quorum:           2
+Flags:            Quorate Qdevice
+
+Membership information
+----------------------
+    Nodeid      Votes    Qdevice Name
+ 3232235777         1    A,V,NMW &subnetI;.1 (local) <co xml:id="co-quorum-nodes"/>
+ 3232235778         1    A,V,NMW &subnetI;.2 <xref linkend="co-quorum-nodes"/>
+         0          1            Qdevice</screen>
+         <calloutlist>
+            <callout arearefs="co-quorum-cmd">
+               <para>
+                  As an alternative with an identical result, you can also use
+                  the <command>corosync-quorumtool</command> command.
+               </para>
+            </callout>
+            <callout arearefs="co-quorum-nodesnumber">
+               <para>
+                  The number of nodes we are expecting. In this example, it is a
+                  two-node cluster.
+               </para>
+            </callout>
+            <callout arearefs="co-quorum-nodeid">
+               <para>
+                  In this case, since it is not explicitly specified in
+                  <filename>corosync.conf</filename>, this ID is an 32-bit integer
+                  representation of the IP address. In this example, the value
+                  <literal>3232235777</literal> stands for the IP address <systemitem
+                     class="ipaddress">&subnetI;.1</systemitem>.
+               </para>
+            </callout>
+            <callout arearefs="co-quorum-quorate">
+               <para>
+                  The quorum status. In this case, the cluster has quorum.
+               </para>
+            </callout>
+            <callout arearefs="co-quorum-nodes">
+               <para>
+                  The status for of each cluster nodes means:
+               </para>
+               <variablelist>
+                  <varlistentry>
+                     <term><literal>A</literal> (Alive) or <literal>NA</literal> (not alive)</term>
+                     <listitem>
+                        <para>Shows the status between &qdevice; and &corosync;.
```

Comment:
Not clear to me, sorry. Should it be: 'Shows the _connection_ status between...'? Or 'Shows if the connection between &qdevice; and &qnet; is still alive (<literal>A</literal>) or not alive (<literal>NA</literal>).'

Reply:
Description from RHEL doc as a reference:

"A/NA — The quorum device is alive or not alive, indicating whether there is a heartbeat between qdevice and corosync. This should always indicate that the quorum device is alive."

Reply:
I've rephrased it to:

    Shows the connection between &qdevice; and &corosync;.
    If it is alive, there is a heartbeat between &qdevice; and &corosync;.

Reply:
How about:
 "Shows the connectivity status between &qdevice; and &corosync;."

And it should be:

"If there is a heartbeat between &qdevice; and &corosync;, it's shown as alive (A)".

Reply:
Great, I use your proposal, much better! :+1: